syndic.opam

```diff
@@ -18,15 +18,24 @@ build: [
   [ "dune" "runtest" "-p" name ] {with-test}
 ]
 
+
 depends: [
   "ocaml" {>= "4.03.0"}
   "dune"
   "ptime"
   "uri" {>= "1.9"}
   "xmlm" {>= "1.2.0"}
-  "cohttp" {with-test & >= "1.0.0"}
-  "cohttp-lwt-unix" {with-test & >= "1.0.0"}
-  "lwt" {with-test}
-  "ssl" {with-test}
-  "tls" {with-test}
+  "fmt" {with-test}
+  "ocurl" {with-test}
+  "fpath" {with-test}
+  "ocplib-json-typed" {with-test}
+  "base-unix" {with-test}
+  "jsonm" {with-test}
 ]
+
+extra-source "test/001.feed" { src: "http://16andcounting.libsyn.com/rss" }
```

Comment:
is this downloaded even when the tests don't run? I am not sure that's the expected behaviour /cc @AltGr

Reply:
Indeed, this will always be downloaded. The `extra-source` section doesn't allow filters at the moment...

Reply:
:(